src/types/dm_channel.rs

```diff
@@ -0,0 +1,19 @@
+use crate::types::User;
+
+#[derive(Deserialize, Clone)]
+pub struct DMChannel {
+    /// The channel ID
+    ///
+    /// 128 bit unsigned integer
+    pub id: u128,
```

Comment:
Also still need to manually implement Serialize for this type. It *should* be a easy copy+paste since Vec<User> is still easy.

Reply:
alright will do